docker/Dockerfile

```diff
@@ -3,18 +3,22 @@ MAINTAINER Serenata de Amor "datasciencebr@gmail.com"
 
 USER root
 
-RUN apt-get update && apt-get install -y \
-  unzip
+RUN apt-get update \
+    && apt-get install unzip -y --no-install-recommends \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
 
 USER jovyan
 
 COPY requirements.txt ./
```

Comment:
BTW, the idea behind splitting up the copy & installs here is to leverage Docker caching mechanism by making sure we don't reinstall conda packages because of python dependencies changes